app/views/index.scala.html

```diff
@@ -178,8 +178,19 @@
             <div class="im-centered3">
                 <div class="row" style="width: 450px;">@Messages("landing.ml.title")</div>
                 <div class="row" id="content-text-4"><p>@Messages("landing.ml.content")</p></div>
-                <br>
+            </div>
+            <br>
+            <div class="gif-container">
                 <img id="mlgif" src="@routes.Assets.at("assets/MLGraphic.gif")">
+                <p class="you">@Messages("landing.ml.gif.you")</p>
```

Comment:
Okay there is some reorganizing of your CSS that should happen at this point because there is a lot of duplicated CSS. What I would probably do is to make a couple CSS that include shared CSS across multiple HTML elements. For example, I imagine that the "user labels" and "machine labels" text have a lot of shared CSS. And the "you", "machine learning", and "outcomes" elements probably have a lot of shared CSS.

You could take what is shared between the three bits of text on the bottom of the GIF, put them into a class called "ml-gif-header-1", and have them each use that class. There are probably three different sets of elements that have shared CSS like that. And maybe there is something shared by all of the text in the GIF, and that could be some CSS that you add to the `gif-container` class, for example.

The reason to do this is in the case when you want to change something, you will want to change it for all of those similar kinds of text. So it's best to not have the styles in multiple places! It allows you to change it once and avoid forgetting about any of them, which could lead to inconsistencies.

Then when you want to deal with the CSS that is very specific to a given element, like the exact positioning, you can give the HTML element an `id`, and add CSS for that specific ID! When making the IDs, I would also go with more informative/descriptive names than "bcp", "btt", "ivt", etc. They should give another coder a better sense of what it actually is. I think that there are a decent number of good examples in `index.scala.html` that shows how to make those IDs :)